impl/src/planning.rs

```diff
@@ -412,18 +435,29 @@ fn load_and_dedup_licenses(licenses: &str) -> Vec<LicenseData> {
   license_data_list
 }
 
+fn package_git_root(manifest: &PathBuf) -> CargoResult<PathBuf> {
+  let cloned = manifest.clone();
+  let mut check_path = cloned.as_path();
+  while let Some(c) = check_path.parent() {
+    let joined = c.join(".git");
+    if joined.is_dir() {
+      return Ok(c.to_path_buf());
+    } else {
+      check_path = c;
+    }
+  }
+
+  Err(CargoError::from(format!("Unable to locate git repository root for manifest {:?}", manifest)))
```

Comment:
Add comment above line
```
// Reached filesystem root and did not find git repo
```

It took me a bit to understand what the exit condition of the above loop was.